lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -295,6 +302,24 @@ readDeployConfig :: MonadObelisk m => FilePath -> FilePath -> m String
 readDeployConfig deployDir fname = liftIO $ do
   fmap (T.unpack . T.strip) $ T.readFile $ deployDir </> fname
 
+lookupKnownHosts :: MonadObelisk m  => String -> m [BS.ByteString]
+lookupKnownHosts hostName =
+  fmap filterComments $ readCreateProcessWithExitCode $ proc "ssh-keygen" ["-F", hostName]
```

Comment:
We'll need to either `staticWhich` this. But do we *need* to use the host's version of this? Thinking...